File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F06.P1.go
package main

import "fmt"

func probe() int {
	// BEGIN PROBE F06.P1
	mid := func(s []int) int { return s[1] }
	xs := []int{1, 2, 3}
	y := mid(xs)
	return y
	// END PROBE F06.P1
}

func main() {
	fmt.Println(probe())
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F06.P1.kt
fun caller(): Int {
    // BEGIN PROBE F06.P1
    fun mid(v: MutableList<Int>) = v[1]
    val xs = mutableListOf(1, 2, 3)
    val y = mid(xs)
    return y
    // END PROBE F06.P1
}

fun main() {
    println(caller())
}
